sample_apps/simple_app_under_sev/instructions.md

```diff
@@ -99,16 +140,16 @@ $CERTIFIER_PROTOTYPE/utilities/simulated_sev_key_generation.exe            \
          --ark_der=sev_ark_cert.der                                        \
          --ask_der=sev_ask_cert.der                                        \
          --vcek_der=sev_vcek_cert.der                                      \
-        --vcek_key_file=$CERTIFIER_PROTOTYPE/src/ec-secp384r1-pub-key.pem
+         --vcek_key_file=/etc/certifier-snp-sim/ec-secp384r1-pub-key.pem
 
-mv ark_cert.der ark_cert.der.old
-mv ask_cert.der ask_cert.der.old
-mv vcek_cert.der vcek_cert.der.old
-mv sev_ark_cert.der ark_cert.der
-mv sev_ask_cert.der ask_cert.der
-mv sev_vcek_cert.der vcek_cert.der
+# Save existing cert.der files as .old files
+$ for der in ark_cert.der ask_cert.der vcek_cert.der; do cp -p ${der} ${der}.old; done
+
+# cp over newly created sev*.der files as cert.der files
+$ for der in ark_cert.der ask_cert.der vcek_cert.der; do cp -p sev_${der} ${der}; done
 ```
 
+# RESOLVE: This comment seems wrong ... fix it.
```

Comment:
@yelvmw -- Let's discuss and fix this text. Suggest what I need to do to reconcile the docs with what's reality.